src/main/java/com/griefprevention/util/command/MonitoredCommands.java

```diff
@@ -157,22 +158,24 @@ private void addCommand(@NotNull Command command, @NotNull String suffix, @Nulla
-         * In the event of a conflict, aliases are not registered and are removed from activeAliases, the
-         * list returned by Command#getAliases. However, the prefixed copy of the alias IS registered.
-         * Conflicting aliases that were prefixed and are active with that prefix can only be detected by
-         * obtaining the original list.
-         * There are a couple options for obtaining the original alias list.
-         * For standard API plugins, the alias list can be obtained from the PluginDescriptionFile.
-         * For other commands, this does not work. To obtain the prefixed aliases (and not leave behind a mess):
-         * 1) The field can be obtained directly with reflection. This is "safer" than getting the command map
-         *   (which is also largely held to be safe) because it is part of the API, which makes it a lot easier
-         *   to check between updates.
-         * 2) A copy of the field can be obtained by obtaining a copy of the current active aliases, telling the
-         *   command it is unregistered using the command map, telling the command it has been re-registered,
-         *   copying the new active aliases (which is a copy of the original aliases after unregistering) and
-         *   removing any entries that were removed from the original active aliases during real registration.
+         * Commands are registered with a fallback prefix. A prefixed version is always
+         * forcibly registered immediately and is not tracked by the command. In the
+         * event of a conflict, aliases are not registered and are removed from
+         * activeAliases, the list returned by Command#getAliases. However, the prefixed
+         * copy of the alias IS registered. Conflicting aliases that were prefixed and
+         * are active with that prefix can only be detected by obtaining the original
+         * list. There are a couple options for obtaining the original alias list. For
+         * standard API plugins, the alias list can be obtained from the
+         * PluginDescriptionFile. For other commands, this does not work. To obtain the
+         * prefixed aliases (and not leave behind a mess): 1) The field can be obtained
+         * directly with reflection. This is "safer" than getting the command map (which
+         * is also largely held to be safe) because it is part of the API, which makes
+         * it a lot easier to check between updates. 2) A copy of the field can be
+         * obtained by obtaining a copy of the current active aliases, telling the
+         * command it is unregistered using the command map, telling the command it has
+         * been re-registered, copying the new active aliases (which is a copy of the
+         * original aliases after unregistering) and removing any entries that were
+         * removed from the original active aliases during real registration.
```

Comment:
I think my only gripe is that auto-formatting comments makes them a lot less readable. It doesn't appear to respect line breaks or extra spacing that helped a lot in this case, though that might just be because the original was too long.